package.json

```diff
@@ -4,17 +4,17 @@
   "private": true,
   "dependencies": {
     "@ant-design/charts": "^1.3.3",
-    "@craco/craco": "^6.4.3",
+    "@craco/craco": "^5.9.0",
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
     "antd": "^4.18.0",
     "axios": "^0.24.0",
-    "craco-less": "^2.0.0",
+    "craco-less": "^1.9.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.2.1",
-    "react-scripts": "5.0.0",
+    "react-scripts": "^3.0.1",
```

Comment:
changing the versions here is a bit dangerous, remind me what was the problem here ?